src/main/java/ua/edu/internship/interview/service/mapper/BaseMapper.java

```diff
@@ -0,0 +1,11 @@
+package ua.edu.internship.interview.service.mapper;
+
+import org.bson.types.ObjectId;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BaseMapper {
+    public String map(ObjectId id) {
```

Comment:
Add unit tests. You should add tests when using some custom logic in mapstruct

Reply:
Added unit tests on mappers. Resolved